src/Magnum/ImGuiIntegration/Widgets.h

```diff
@@ -55,7 +55,7 @@ an implementation detail that might change in the future.
 @see @ref image(), @ref imageButton()
 */
 inline ImTextureID textureId(GL::Texture2D& texture) {
-    return reinterpret_cast<ImTextureID>(texture.id());
+    return static_cast<ImTextureID>(texture.id());
```

Comment:
Could you make it so it still works on older versions? I suppose it'd be something like this -- and given that both types are now unsigned ints, it should work without any explicit cast:

```suggestion
    #if IMGUI_VERSION_NUM >= 19131
    return texture.id();
    #else
    return reinterpret_cast<ImTextureID>(texture.id());
    #endif
```